src/adapters/Base/index.test.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable camelcase */
+import requestManager from 'lib/requestManager';
+
+import baseAdapter from '.';
+
+jest.mock('lib/requestManager');
+
+describe('BaseAdapter', () => {
+  const apiPath = '/sample';
+  const params = {
+    testKey: 'test value',
+  };
+
+  beforeEach(() => {
+    (requestManager as jest.Mock).mockImplementation(() => jest.fn());
```

Comment:
does it work?
```suggestion
    jest.mock('lib/requestManager', () => jest.fn())
```
FYI, the mock path works with relative path only